### PR TITLE
If /me rejects with an error, throw this error rather than setting pr…

### PR DESCRIPTION
**Pull request by joefitter**
…ofile to null

### CODE REVIEW

File: lib/auth/profile.js [hunk 1]
```diff
@@ -34,7 +34,9 @@ module.exports = (endpoint) => {
               allowedActions: meta.allowedActions
             };
           })
-          .catch(() => null);
+          .catch(err => {
+            throw err;
+          });
```
Comment (lennym) [inline lib/auth/profile.js]: This is a total no-op. Just remove the `catch` block completely.

Comment (joefitter) [reply lib/auth/profile.js]: herr derp

**Comment by lennym** [comment]
LG...

...TM

**Comment by joefitter** [comment]
Lenny goes... to market?

**Pull request merged**